docs/2023/rest/updates/soham/2023-06-01.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Week 1
+author: Soham Banerjee
+author_url: https://github.com/soham4abc
+author_image_url: https://avatars.githubusercontent.com/u/63705023?v=4
+tags: [gsoc23, rest]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2023 Soham Banerjee <sohambanerjee4abc@hotmail.com>
+-->
+
+## Activities
+
+- Issued Pull Request for the OpenAPI endpoint. `/openapi`: [#2474](https://github.com/fossology/fossology/pull/2474).
+
+### SOME SCREENSHOTS
```

Comment:
```suggestion
### SOME SCREENSHOTS
```
Is this some format decided by you all? 😅 cc @dushimsam, apologies if I am not aware

Reply:
No actually.... I tried to follow his pattern thinking that it was the standard as it was merged... :-)

Reply:
Should I change it?? @avinal

Reply:
No, it is fine, it doesn't make much difference as long as it is correct information.

Reply:
Thank you @avinal !!